operator/api/v1alpha2/keptnworkloadinstance_types.go

```diff
@@ -63,24 +63,15 @@ type KeptnWorkloadInstanceStatus struct {
 	Status common.KeptnState `json:"status,omitempty"`
 }
 
-type TaskStatus struct {
-	TaskDefinitionName string `json:"taskDefinitionName,omitempty"`
+type ItemStatus struct {
+	DefinitionName string `json:"definitionName,omitempty"`
 	// +kubebuilder:default:=Pending
 	Status    common.KeptnState `json:"status,omitempty"`
-	TaskName  string            `json:"taskName,omitempty"`
+	Name      string            `json:"name,omitempty"`
```

Comment:
maybe it makes sense to add a docs comment to make the difference between DefinitionName and Name more clear?